shapeshift/web
# chore: lifi off, polygon on

## Description



* disables li.fi for all swap/bridge routes
* turns polygon base support and swapping on for all environments

## Pull Request Type


## Issue (if applicable)



n/a

## Risk

low - polygon tested in develop by ops, turning flag off for li.fi



## Testing

* see that polygon is on
* see that lifi is off



### Engineering

👆



### Operations

👆



## Screenshots (if applicable)
